### PR TITLE
Fix successful login even if password incorrect

### PR DESCRIPTION
**Pull request by jdeepee**

### CODE REVIEW

File: src/views/welcome-view/WelcomeView.vue [hunk 1]
```diff
@@ -236,7 +239,17 @@ export default defineComponent({
         .dispatch("logIn", {
           password: this.password,
         })
-        .then(() => this.$router.push("/"))
+        .then(async () => {
```
Comment (leifriksheim) [inline src/views/welcome-view/WelcomeView.vue]: Don't need async here as there is no promise being executed here.

Comment (leifriksheim) [reply src/views/welcome-view/WelcomeView.vue]: @jdeepee Will the UNLOCK_AGENT mutation throw an error if the password is wrong?

**Review by leifriksheim — APPROVED** [review]
Made a small change where we just check if the user response from the action is unlocked

File: src/views/welcome-view/WelcomeView.vue (hunk 1 above)
Comment (jdeepee) [reply src/views/welcome-view/WelcomeView.vue]: @leifriksheim no it will just return isUnlocked = false in the response

**Pull request merged**